modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -177,7 +177,7 @@ def test_model(self):
             cv.rectangle(frame, box, (0, 255, 0))
             cv.rectangle(frame, np.array(box), (0, 255, 0))
             cv.rectangle(frame, tuple(box), (0, 255, 0))
-            cv.rectangle(frame, list(box), (0, 255, 0))
+            # FIXIT never properly work: cv.rectangle(frame, list(box), (0, 255, 0))
```

Comment:
Disabled, because conversion to Rect is never worked properly here (result of `Rect(0,0,0,0)` or garbage due to index out of range access).

(conversion should be rewritten, starting form "3.4" branch with related tests - out of scope this hotfix)